csharp_dotnetcore/1.Console-EchoBot/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Console_EchoBot
+{
+    using System;
+    using System.Threading;
+
+    /// <summary>
+    /// Main program for EchoBot.
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main for the Console EchoBot.
+        /// </summary>
+        /// <param name="args">Arguments passed to the program.  Ignored.</param>
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to the EchoBot. Type something to get started.");
+
+            // Create the Console Adapter, and add Conversation State
+            // to the Bot. The Conversation State will be stored in memory.
+            var adapter = new ConsoleAdapter();
+
+            // Create the instance of our Bot.
+            var echoBot = new EchoBot();
+
+            // Connect the Console Adapter to the Bot.
+            adapter.ProcessActivityAsync(
+                async (context, ct) => await echoBot.OnTurnAsync(context, default(CancellationToken))).Wait();
```

Comment:
@johnataylor, @cleemullins for the .NET bots we should be using `turnContext` and `cancellationToken` in samples as well? Or just in the libraries themselves?

See Microsoft/botbuilder-dotnet#867

Reply:
I don't think this sample should use Cancellation Tokens. For those who want them, they can add them. For most, they're just noise in the code.

Reply:
context => turnContext